Workbooks/Azure Active Directory/EntitlementManagementAppRoleAssignments/EntitlementManagementAppRoleAssignments.workbook

```diff
@@ -0,0 +1,105 @@
+          }
+        ],
+        "style": "pills",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "name": "parameters - 1"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "AuditLogs | where TimeGenerated > ago({Since:value}) and (Category == \"UserManagement\" or Category == \"GroupManagement\" or Category == \"ApplicationManagement\") and Result == \"success\" and (ActivityDisplayName == \"Add app role assignment grant to user\" or ActivityDisplayName == \"Add app role assignment to group\" or ActivityDisplayName == \"Add app role assignment to service principal\" or ActivityDisplayName == \"Add a deletion-marked app role assignment grant to user as part of link removal\" or ActivityDisplayName == \"Add a deletion-marked app role assignment grant to group as part of link removal\" or ActivityDisplayName == \"Add a deletion-marked app role assignment grant to service principal as part of link removal\" or ActivityDisplayName == \"Remove app role assignment from user\" or  ActivityDisplayName == \"Remove app role assignment from group\" or ActivityDisplayName == \"Remove app role assignment from service principal\") and (InitiatedBy.app.displayName != \"{ExcludedInitiatorAppName:value}\" or InitiatedBy.app.appid != \"\") | extend ByUPN = InitiatedBy.user.userPrincipalName,ByServicePrincipal=InitiatedBy.app.servicePrincipalName,ByAppId=InitiatedBy.app.appid,ByApp=InitiatedBy.app.displayName |project ActivityDateTime,ActivityDisplayName,ByUPN,ByAppId,ByApp,ByServicePrincipal,TargetResources | mv-expand TargetResources | extend DateTime=ActivityDateTime,App=TargetResources.displayName,MP=TargetResources.modifiedProperties | where MP != \"[]\" and App == \"{Appname:value}\" | mv-apply MPARDN=MP to typeof(dynamic) on ( where MPARDN.displayName==\"AppRole.DisplayName\" ) | mv-apply MPUUPN=MP to typeof(dynamic) on ( where MPUUPN.displayName ==\"User.UPN\" or MPUUPN.displayName==\"Group.DisplayName\") | mv-apply MPUO=MP to typeof(dynamic) on ( where MPUO.displayName ==\"User.ObjectID\" or MPUO.displayName ==\"Group.ObjectID\") | extend AppRoleName=trim(\"\\\"\",tostring(MPARDN.newValue)),Subject=trim(\"\\\"\",tostring(MPUUPN.newValue)),SubjectId=trim(\"\\\"\",tostring(MPUO.newValue)) |project DateTime,ActivityDisplayName,ByUPN,ByServicePrincipal,ByApp,ByAppId,App,AppRoleName,Subject,SubjectId | order by DateTime desc",
+        "size": 0,
+        "title": "Recent app role assignments to users and groups",
+        "noDataMessage": "No app role assignment updates found in the specified time period",
+        "noDataMessageStyle": 3,
+        "showExportToExcel": true,
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "gridSettings": {
```

Comment:
to get localizable column names, go into the grid settings, and click on the "labels" tab, and set labels on each column.  column labels set there will get localized, without them, you'll get the names exactly as returned

Reply:
added column labels